### PR TITLE
Updated portal guidance

### PR DESCRIPTION
**Pull request by kjyam98**
@danieldobalian  portal guidance updated, please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,20 +61,21 @@ You will need to have a native client application registered with Microsoft usin
...
 
 2. Create the app
-    - Click ***New application registration***.  
-    - Enter an app name in the ***Name*** field. 
-    - In ***Application type***, select `Native`. 
-    - In ***Redirect URI***, enter `http://localhost`.  
+    - Select ***New registration***.  
+    - Enter an application name in the ***Name*** field. 
+    - Set ***Supported account types*** to ***Accounts in any organizational directory***. 
+    - Leave ***Redirect URI*** empty.
```
Comment (danieldobalian) [inline README.md]: I think this sample requires the developer to register `http://localhost` in order to run.

File: README.md [hunk 2]
```diff
@@ -61,20 +61,21 @@ You will need to have a native client application registered with Microsoft usin
...
 
 3. Configure Microsoft Graph
-    - Select ***Settings*** > ***Required Permissions***.
-    - Click ***Add***, inside ***Select an API*** select ***Microsoft Graph***. 
-    - Select the permission `Sign in and read user profile` > Hit `Select` to save. 
-        - This permission maps to the `User.Read` scope. 
+    - Under ***Manage***, select ***API permissions***.
+    - Click ***Add a permission***, inside ***Request API permissions*** select ***Microsoft Graph***. 
+    - Select the permission type ***Delegated permissions***
+    - Under ***Select Permissions*** type `user.read` which will select the appropriate scope.
```
Comment (danieldobalian) [inline README.md]: OAuth2 calls for scopes to be case sensitive - I'm not sure if AAD enforces that, however. Portal/v2 sample both use `User.Read`, should probably follow that convention.

File: README.md (hunk 1 above)
Comment (kjyam98) [reply README.md]: Thanks - I made your change.

File: README.md (hunk 2 above)
Comment (kjyam98) [reply README.md]: Thanks - I made your change.

**Comment by kjyam98** [comment]
@danieldobalian fixes made - please review.

File: README.md [hunk 3]
```diff
@@ -61,20 +61,21 @@ You will need to have a native client application registered with Microsoft usin
...
 
 2. Create the app
-    - Click ***New application registration***.  
-    - Enter an app name in the ***Name*** field. 
-    - In ***Application type***, select `Native`. 
-    - In ***Redirect URI***, enter `http://localhost`.  
+    - Select ***New registration***.  
+    - Enter an application name in the ***Name*** field. 
+    - Set ***Supported account types*** to ***Accounts in any organizational directory***. 
+    - Leave ***Redirect URI*** `http://localhost`.
```
Comment (danieldobalian) [inline README.md]: Recommend the following for this:

- Set the ***Redirect URI***  field to `Public client` and add the value `http://localhost`.

Comment (kjyam98) [reply README.md]: @danieldobalian updated the Redirect URI guidance - please review.

**Pull request closed without merging**